rectools/visuals/visual_app.py

```diff
@@ -0,0 +1,273 @@
+import typing as tp
+
+import ipywidgets as widgets
+import numpy as np
+import pandas as pd
+from IPython.display import display
+
+from rectools import Columns
+
+
+class AppDataStorage:
+    """
+    Storage and processing of data for `VisualApp` widgets. This class is not meant to be used
+    directly. Use `VisualApp` class instead
+    """
+
+    def __init__(
+        self,
+        recos: tp.Dict[tp.Hashable, pd.DataFrame],
+        item_data: pd.DataFrame,
+        requests_dict: tp.Dict[tp.Hashable, tp.Hashable],
```

Comment:
you don't use `_dict` part in `recos` argument, but use here

I think we can remove it

Reply:
fixed